### ISSUE TITLE
Add rule for passing prop that doesn't exist

### ISSUE THREAD

**Issue by benmccann** [sveltejs/eslint-plugin-svelte#1156]
### Description

I noticed I have some code that looks like this, but where I've deleted `onClose`:
```
        <SlideshowBar
          {isFullScreen}
          onSetToFullScreen={() => assetViewerHtmlElement?.requestFullscreen?.()}
          onPrevious={() => navigateAsset('previous')}
          onNext={() => navigateAsset('next')}
          onClose={() => ($slideshowState = SlideshowState.StopSlideshow)}
        />
```

There's nothing that tells me I'm passing a prop that doesn't exist

**Comment by baseballyama** [comment]
Shouldn’t this be detected by type checking?

**Comment by benmccann** [comment]
Oops. Right you are! `svelte-check` catches it. It wasn't showing up in my IDE for some reason, but it's catching it from the CLI
